.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214235900-d7faa5e7-6e3b-45a1-a72b-eee68bdb9597.png)

- [ ] You didn't acknowledge anyone in your readme file. :fearful: Kindly add Microverse in this section to acknowledge Microverse :+1:

Reply:
I will add it right away sir thank you